### ISSUE TITLE
Disable "Default and local language name not the same" in bilingual-region

### ISSUE THREAD

**Issue by ivanbranco** [osm-fr/osmose-backend#1645]
Hi, 

item | 5060
class | 50601
example | https://osmose.openstreetmap.fr/it/issue/67819af9-4a46-77a1-4b93-64fdfe4c2cbb

would be possible to exclude [italy_friuli_venezia_giulia](https://osmose.openstreetmap.fr/it/issues/open?country=italy_friuli_venezia_giulia) from the ["Default and local language name not the same"](https://osmose.openstreetmap.fr/it/issues/open?source=2560&item=5060&class=50601) check?

This region is [bilingual](https://it.wikipedia.org/wiki/Bilinguismo_amministrativo_in_Italia#Friuli-Venezia_Giulia) and most of the names are in italian-slovenian, which produces a lot of false positive errors in Osmose.

**Comment by frodrigo** [comment]
We can switch to multilingual check https://github.com/osm-fr/osmose-backend/blob/master/plugins/Name_Multilingual.py

There is already multiple multilingual variant implementations.

Have you the ability to check if on variant implemented is already usable or need a new one ?

Do you have doc about how names are mapped in this area?

**Comment by ivanbranco** [comment]
> Have you the ability to check if on variant implemented is already usable or need a new one ?

Unfortunately no.

> Do you have doc about how names are mapped in this area?

the consensus is `"italian name / slovenian name". An example: https://www.openstreetmap.org/node/5282009899

**Comment by ivanbranco** [comment]
I just wanted to check in and see if there’s any additional information needed for this issue. If so, I’d be happy to help. If I’m unable to provide the details myself, I can open a thread in the Italian community forum to gather more input.

Thanks!

**Comment by frodrigo** [comment]
If the order is fixed, (italian name / slovenian name) it should be implemented, else it is already available.
https://github.com/osm-fr/osmose-backend/blob/master/plugins/Name_Multilingual.py#L48-L75

**Comment by ivanbranco** [comment]
> If the order is fixed, (italian name / slovenian name)

Yes, is fixed and documented in the Italian wiki ([here](https://wiki.openstreetmap.org/wiki/Multilingual_names#Friuli-Venezia_Giulia)):

> There seems to be [consensus](http://overpass-turbo.eu/s/V7J) in including italian and slovenian languages in the general name-tag separated by a slash, where the italian name comes first.
> 
> Examples:
> 
> [name](https://wiki.openstreetmap.org/wiki/Key:name)=Duino / Devin
> [name:sl](https://wiki.openstreetmap.org/wiki/Key:name:sl)=Devin
> [name:it](https://wiki.openstreetmap.org/wiki/Key:name:it)=Duino
